src/components/Card/Card.test.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { cardsData } from '../../api/cardsData';
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import Card from './Card';
+
+describe('when rendered', () => {
+  const productStab = cardsData.products[0];
+  const { title, thumbnail, price, description, rating, brand, id } = productStab;
+  it('Should have product title', () => {
+    render(
+      <Card
+        id={id}
+        title={title}
+        brand={brand}
+        description={description}
+        price={price}
+        rating={rating}
+        thumbnail={thumbnail}
```

Comment:
You can put these props into the object and use `<Card {...productProps} />` syntax

Reply:
fix